cocos/2d/CCMotionStreak.cpp

```diff
@@ -126,7 +127,7 @@ bool MotionStreak::initWithFade(float fade, float minSeg, float stroke, const Co
     _blendFunc = BlendFunc::ALPHA_NON_PREMULTIPLIED;
 
     // shader state
-    setGLProgramState(GLProgramState::getOrCreateWithGLProgramName(GLProgram::SHADER_NAME_POSITION_TEXTURE_COLOR));
+    setGLProgramState(GLProgramState::getOrCreateWithGLProgramName(GLProgram::SHADER_NAME_POSITION_TEXTURE_A8_COLOR));
```

Comment:
I don't fully understand your problem here, the texture color might be ignored but is that necassarily a bad thing? For me personally I can't even notice it since the texture is just a white square.

Besided that, introducing a new shader which solves this is an easy fix.